### PR TITLE
Completing files after completing the difference task

### PR DESCRIPTION
**Pull request by vaibhavNagvekarSJI**

### CODE REVIEW

File: BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+		if (eleDisplayed) {
+			extentPassLog("Related Content header displayed : ", true);
+			String actualHeader = relatedContentHeader.getText();
+			Action.printAndAssert(actualHeader, headerTitle);
+			Action.explicitWait(articleName(article), 30);
+			String actualArticle = articleName(article).getText();
+			Action.printAndAssert(actualArticle, article);
+			Action.explicitWait(readMoreLink(readMore), 30);
+			readMoreLink(readMore).click();
+			extentInfoLog("Clicked on : ", "READ MORE link");
```
Comment (goutamnaik007) [inline BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java]: replace the click and log with "Action.performActionwithExtentInfoLog"
@vaibhavNagvekarSJI

File: BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+		Action.explicitWait(articleName(heading), 30);
+		boolean eleDisplayed = articleName(heading).isDisplayed();
+		if (eleDisplayed) {
+			extentPassLog("Related Products header displayed : ", true);
+			String actualArticle = articleName(heading).getText();
+			Action.printAndAssert(actualArticle, heading);
+			Action.explicitWait(productName(product), 30);
+			productName(product).click();
+			extentInfoLog("Clicked on : ", product);
```
Comment (goutamnaik007) [inline BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java]: replace the click and log with "Action.performActionwithExtentInfoLog"
@vaibhavNagvekarSJI

File: BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+			Action.verifyPageUrl(expectedUrl);
+			Action.navigateBack();
+			extentInfoLog("Navigating back to : ", "Difference page");
+			Action.explicitWait(getAges(product), 30);
+			String actualAges = getAges(product).getText();
+			Action.printAndAssert(actualAges, expectedAges);
+			Action.explicitWait(buyNowBtn(product), 30);
+			buyNowBtn(product).click();
+			extentInfoLog("Clicked on : ", "buy now button");
```
Comment (goutamnaik007) [inline BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java]: replace the click and log with "Action.performActionwithExtentInfoLog"
@vaibhavNagvekarSJI

Comment (vaibhavNagvekarSJI) [reply BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java]: fixed

File: BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java (hunk 2 above)
Comment (vaibhavNagvekarSJI) [reply BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java]: fixed

File: BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java (hunk 1 above)
Comment (vaibhavNagvekarSJI) [reply BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java]: fixed

Comment (goutamnaik007) [reply BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java]: Verified @vaibhavNagvekarSJI

File: BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+			Action.verifyPageUrl(expectedUrl);
+			Action.navigateBack();
+			extentInfoLog("Navigating back to : ", "Difference page");
+			Action.explicitWait(getAges(product), 30);
+			String actualAges = getAges(product).getText();
+			Action.printAndAssert(actualAges, expectedAges);
+			Action.explicitWait(buyNowBtn(product), 30);
+			Action.performActionwithExtentInfoLog(buyNowBtn(product), "click",
+					"Clicking on : " + buyNowBtn(product).getText());
+			Action.waitFor(5000);
```
Comment (goutamnaik007) [inline BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java:174]: is this required @vaibhavNagvekarSJI

File: BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java (hunk 3 above)
Comment (goutamnaik007) [reply BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java]: Verified @vaibhavNagvekarSJI

File: BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java (hunk 2 above)
Comment (goutamnaik007) [reply BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java]: Verified @vaibhavNagvekarSJI

File: BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java (hunk 4 above)
Comment (vaibhavNagvekarSJI) [reply BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java:174]: yes it needs

**Pull request merged**